includes/class-wcmp-data.php

```diff
@@ -114,6 +114,16 @@ public static function getDeliveryTypeHuman($deliveryType): string
         );
     }
 
+    /**
+     * @param int|string $deliveryType
```

Comment:
kan dit een id zijn en een string?

Reply:
In de oude situatie kon het int en string zijn. nu alleen maar string

Reply:
Done